mailbox/backup/src/main/java/org/apache/james/mailbox/backup/FlagsExtraField.java

```diff
@@ -0,0 +1,73 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.mailbox.backup;
+
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import javax.mail.Flags;
+
+import org.apache.commons.compress.archivers.zip.ZipShort;
+import org.apache.james.util.StreamUtils;
+
+public class FlagsExtraField extends StringExtraField {
+
+    public static final ZipShort ID = new ZipShort(0x7061); // "ap" in little-endian
```

Comment:
the ZipShort should be named "ap" then, don't you think ?

Reply:
I don't understand what you mean, I follow the rule of other ZipExtraField in James before.

Reply:
`ID_AP` can be cool yes.

Why not doing the same changes everywhere in this module as a separate commit?

Reply:
I changed ID for this and other ZipExtraField